.gitignore

```diff
@@ -1,4 +1,4 @@
 /.phpunit.result.cache
 /composer.lock
 /phpunit.xml
-/vendor
+/vendor
```

Comment:
This diff should be reverted

Reply:
Vendor is part of the original [.gitignore](https://github.com/gitonomy/gitlib/blob/1.3/.gitignore) though? @lyrixx

Reply:
yes, but I mean the diff about the trailing \n ; you removed it
![image](https://user-images.githubusercontent.com/408368/183038710-3add0e3f-add7-420d-904e-aafa945cc831.png)

Reply:
Ahhh I see - my IDE seems to remove it automatically - is it necessary to add it? If yes, then I'll have to look it up how to change my IDE to not delete seemingly unnecessary trailing \ns

> @lyrixx  yes, but I mean the diff about the trailing \n ; you removed it ![image](https://user-images.githubusercontent.com/408368/183038710-3add0e3f-add7-420d-904e-aafa945cc831.png)

Reply:
Okay i figured it out, it should be good now :)